tasks/utils.py

```diff
@@ -78,6 +79,10 @@ def get_build_flags(ctx, static=False, prefix=None, embedded_path=None,
     if python_home_3:
         ldflags += "-X {}/pkg/collector/python.pythonHome3={} ".format(REPO_PATH, python_home_3)
 
+    # If we're not building with both Python, we want to force the use of Python 3
+    if not with_both_python:
+      ldflags += "-X {}/pkg/config.ForceDefaultPython=3 ".format(REPO_PATH)
```

Comment:
`ForceDefaultPython` should be a boolean I think (maybe stored as a string if only string values are allowed, but still have the meaning of a boolean).

reason: `DefaultPython` already defines what version is the default. I think `ForceDefaultPython` only needs to define if we should force that version or not. Also, makes the logic here more flexible (if we only shipped with python 2 for some reason, we would want this logic to force python2, which is easy to do if this is just a boolean).

What do you think?

Reply:
SGTM indeed, I've reflected this change.